WooCommerce/Classes/ViewRelated/Blaze/AdDestination/BlazeAdDestinationSettingViewModel.swift

```diff
@@ -79,15 +126,15 @@ private extension BlazeAdDestinationSettingViewModel {
         enum parameterCharactersLimit {
             static let plural = NSLocalizedString(
                 "blazeAdDestinationSettingVieModel.parameterCharactersLimit.plural",
-                value: "%d characters remaining",
-                comment: "Blaze Ad Destination: Plural form for characters limit label %d will be replaced by a number. " +
+                value: "%1$@ characters remaining",
```

Comment:
🚨 This crashes the app because you are using `@` for a digit parameter. Please switch to `d` instead.

Reply:
handled in https://github.com/woocommerce/woocommerce-ios/pull/11642/commits/34b30ce01ae95c9a64f1cbecaa53399ba2a3df23